src/Utils.ts

```diff
@@ -313,34 +313,37 @@ export namespace Utils {
     }
 
     export async function showEffectivePom(pomPath: string): Promise<void> {
-        const pomxml: string = await window.withProgress({ location: ProgressLocation.Window }, (p: Progress<{ message?: string }>) => new Promise<string>(
+        let pomxml: string;
+        const project: MavenProject = mavenExplorerProvider.getMavenProject(pomPath);
+        if (project) {
+            await project.calculateEffectivePom();
+            pomxml = project.rawEffectivePom;
+        } else {
+            pomxml = await Utils.getEffectivePom(pomPath);
+        }
+
+        if (pomxml) {
+            const document: TextDocument = await workspace.openTextDocument({ language: "xml", content: pomxml });
+            window.showTextDocument(document);
```

Comment:
Show in the active ViewColumn?

Reply:
Sounds a good idea. So I suppose by using ViewColumn, I can specify the title of it, e.g. `[Project 1]Effective-POM`?

Reply:
Got it. Updated.